userspace/libsinsp/filterchecks.cpp

```diff
@@ -5874,7 +5888,13 @@ inline uint8_t* sinsp_filter_check_evtin::extract_tracer(sinsp_evt *evt, sinsp_p
 
 		if(m_storage_size < encoded_tags_len)
 		{
-			m_storage = (char*)realloc(m_storage, encoded_tags_len);
+			char *reduced_storage = (char*)realloc(m_storage, encoded_tags_len);
+			if(reduced_storage == NULL)
+			{
+				free(m_storage);
```

Comment:
I think this is a double free on `m_storage`.

Reply:
done! https://github.com/falcosecurity/libs/pull/888/commits/e90f4f3d3e86e07721d8ea5d03d291bb26c8fbfd